ghost/i18n/locales/es/portal.json

```diff
@@ -200,7 +200,7 @@
     "You've successfully signed in.": "Has iniciado sesión correctamente.",
     "You've successfully subscribed to": "Te has suscrito correctamente a",
     "Your account": "Tu cuenta",
-    "Your email has failed to resubscribe, please try again": "",
+    "Your email has failed to resubscribe, please try again": "Tu correo electrónico ha fallado para re suscribirse, intentalo de nuevo por favor",
```

Comment:
as we are using 'tu', shouldn't this be suscribirte ?  Ah, maybe I am misunderstanding the grammar here, and it's the email that is resubscribing, so then -se is appropriate.  Honestly, this sentence is a little weird in English...

Reply:
"Tu" is the same as "You". As in "Your email" means "Tu correo electrónico". "Resuscribe" in proper Spanish is re suscribirse, as resuscribirse is not even a word, "re suscribirse" however is the same as to "resuscribe". It's given the proper syntaxis of the sentence and connotation.

Reply:
That wasn't the question I asked. I asked whether it should have a -se or -te ending, because I thought that the actor here was 'tu' instead of 'correo electronico'.  My mistake, sorry.